packages/reporting-server/setup.py

```diff
@@ -31,6 +33,7 @@
     entry_points={
         "console_scripts": [
             "reporting_server=rest_server.__main__:main",
+            "reporting_server_dev=rest_server.__dev__:main",
```

Comment:
I don't think we should be exporting the dev scripts.

Reply:
Yeah, you're right!